colibris/__init__.py

```diff
@@ -14,7 +14,7 @@
 from colibris import template
 
 
-VERSION = 'unknown'
```

Comment:
This is intentionally set to `unknown` and will automatically be replaced by the CI: https://github.com/colibris-framework/colibris/blob/9655016637888c480f49f92711caa6088013e442/.github/workflows/main.yml#L53

Reply:
I've set it to a valid version placeholder so that I can run `tox` or install it via `pip` without having to run sed.